### ISSUE TITLE
Bevdet_occ bev增强，mIou仅为5.96

### ISSUE THREAD

**Issue by haoyuan1770** [HuangJunJie2017/BEVDet#266]
感谢作者做出的贡献，我利用给出的configs/bevdet_occ/bevdet-occ-r50-4d-stereo-24e.py，

1.仅使用bev增强，关闭图像增强，推理的mIou仅为5.96。配置如下：
data_config = {
    'cams': [
        'CAM_FRONT_LEFT', 'CAM_FRONT', 'CAM_FRONT_RIGHT', 'CAM_BACK_LEFT',
        'CAM_BACK', 'CAM_BACK_RIGHT'
    ],
    'Ncams':
    6,
    'input_size': (256, 704),
    'src_size': (900, 1600),

    # Augmentation
    'resize': (0.0, 0.0),
    'rot': (0.0, 0.0),
    'flip': False,
    'crop_h': (0.0, 0.0),
    'resize_test': 0.00,
}
bda_aug_conf = dict(
    rot_lim=(-0., 0.),
    scale_lim=(1., 1.),
    flip_dx_ratio=0.5,
    flip_dy_ratio=0.5)

2.仅使用图像增强，结果比36.01高。配置如下：
data_config = {
    'cams': [
        'CAM_FRONT_LEFT', 'CAM_FRONT', 'CAM_FRONT_RIGHT', 'CAM_BACK_LEFT',
        'CAM_BACK', 'CAM_BACK_RIGHT'
    ],
    'Ncams':
    6,
    'input_size': (256, 704),
    'src_size': (900, 1600),

    # Augmentation
    'resize': (-0.06, 0.11),
    'rot': (-5.4, 5.4),
    'flip': True,
    'crop_h': (0.0, 0.0),
    'resize_test': 0.00,
}
bda_aug_conf = dict(
    rot_lim=(-0., 0.),
    scale_lim=(1., 1.),
    flip_dx_ratio=0.0,
    flip_dy_ratio=0.0)

请问是否有问题还是我这边设置的不对。

**Comment by HuangJunJie2017** [comment]
@haoyuan1770 It seems ok. Check the loss. Or test the checkpoints in the early stage of training to check the overfitting problem.
